src/main/java/org/folio/rest/impl/TenantPermsAPI.java

```diff
@@ -44,10 +59,10 @@ public void postTenantpermissions(OkapiPermissionSet entity, Map<String, String>
                                     Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
     try {
       String tenantId = TenantTool.tenantId(okapiHeaders);
-      savePermList(entity.getPerms(), vertxContext, tenantId).onComplete(savePermsRes -> {
-        if (savePermsRes.failed()) {
-          String err = savePermsRes.cause().getMessage();
-          logger.error(err, savePermsRes.cause());
+      handlePermLists(entity, vertxContext, tenantId).onComplete(ar -> {
```

Comment:
Replacing onComplete by onFailure and onSucces is more concise. https://github.com/folio-org/raml-module-builder/blob/master/doc/futurisation.md#result-and-error-handling

Reply:
Yup, made this change in several places